doc/users/dflt_style_changes.rst

```diff
@@ -473,6 +473,36 @@ or by setting::
 in your :file:`matplotlibrc` file.
 
 
+``fill_between`` and ``fill_betweenx``
+--------------------------------------
+
+`~matplotlib.axes.Axes.fill_between` and
+`~matplotlib.axes.Axes.fill_betweenx` both follow the patch color
+cycle.
```

Comment:
Is this misleading?  The color cycle is attached to the Axes instance, isn't it?